app/src/main/java/com/example/oddhours/utils/Helper.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.oddhours.utils
+
+class Helper {
```

Comment:
maybe consider giving the helper class a more descriptive name? usually we'd go for something like `JobHelper` or `ShiftHelper` or `RepositoryHelper` ... totally up to you. since the codebase isn't complicated i'm also fine having just one helper class for all utility functions

Reply:
ShiftHelper works. I just called it Helper for now, cuz I wasn't sure how much it'd be growing.

Reply:
ya man no worries. the code base isn't super complicated at the moment so either separating it or leaving it all in one helper class is fine